src/CopyrighFixer/CopyrighFixer/bar.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BAR_H
+#define BAR_H
+
+#include <QString>
+#include <iostream>
+
+class Bar
+{
+public:
+    Bar();
+    QString bar = "Hello WORLD!";
```

Comment:
Why you create a member with string value?
I think if you return the constant literal value in the foo method then is will be more performance solution then this.

Reply:
Done